src/canvas/SelectableCanvas.ts

```diff
@@ -772,17 +771,11 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     obj: FabricObject,
     globalPointer: Point
   ): boolean {
-    if (
-      obj &&
-      obj.visible &&
-      obj.evented &&
-      // http://www.geog.ubc.ca/courses/klink/gis.notes/ncgia/u32.html
-      // http://idav.ucdavis.edu/~okreylos/TAship/Spring2000/PointInPolygon.html
```

Comment:
we should preserve this comment maybe over the containsPoint function

Reply:
both are dead, and also the impl has moved forward more than a decade